lib/entities/organization-membership.js

```diff
@@ -72,16 +82,24 @@ function createOrganizationMembershipApi (http) {
      *   accessToken: '<content_management_api_key>'
      * })
      *
-     * client.getOrganization('<organization_id>')
-     * .then((organization) => organization.getOrganizationMembership('<OrganizationMembership_id>'))
-     * .then((OrganizationMembership) => OrganizationMembership.delete())
-     * .then(() => console.log(`OrganizationMembership deleted.`))
+     * client.getOrganizations()
+     * .then((organizations) => orientations.items[0])
```

Comment:
i think this should be .then((organizations) => organizations.items[0]) :)

Reply:
totally 👍fixed